modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,186 @@
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "ReconcileComplete"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Progressing"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T16:39:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Degraded"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Upgradeable" <1>
+  }
+]
+----
+====
+<1> The {VirtProductName} Operator has the `Upgradeable` status.
+
+. Manually update your cluster from the source EUS version to the next minor version of {product-title}:
++
+[source,terminal]
++
+----
+$ oc adm upgrade
+----
++
+.Verification
+* Check the current version by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+[NOTE]
+====
+Updating {product-title} to the next version is a prerequisite for updating {VirtProductName}. For more details, refer to the "Updating clusters" section of the {product-title} documentation.
+====
+
+. If you use the recommended *Automatic* approval strategy, which is enabled by default, {VirtProductName} automatically updates to the corresponding version after you update {product-title}. Monitor the {VirtProductName} update by running the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-cnv
+----
++
+[NOTE]
+====
+If you used the *Manual* approval strategy, you must approve the update in the web console. For more details, refer to the "Manually approving a pending Operator update" section.
+====
+
+. Confirm that {VirtProductName} successfully updated to the non-EUS version by running the following command:
++
+[source,terminal]
+----
+$ oc get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.versions"
+----
++
+.Example output
+[source,terminal]
+----
+[
+  {
+    "name": "operator",
+    "version": "{HCOVersion}"
+  }
+]
+----
+
+. Wait until the `HyperConverged` Operator has the `Upgradeable` status before you perform the next update. Enter the following command and monitor the output:
++
+[source,terminal]
+----
+$ oc get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.conditions"
+----
+
+. Continue to update your cluster and {VirtProductName} as described in the preceding steps, monitoring to ensure that the updates succeed.
```

Comment:
`until you install the latest z-stream release of the non-EUS minor version.` this is relevant to the line above (157). At this point (line 158) we have already reached the list z-stream of non-EUS, now we need to upgrade OCP and final Y-stream to EUS. `Ensure that you update both {product-title}`. We only need two OCP upgrades. One from source EUS-> non-EUS and then non-EUS->EUS target.